functions/Test-DbaSqlLoginWeakPassword.ps1

```diff
@@ -0,0 +1,131 @@
+function Test-DbaSqlLoginWeakPassword {
+    <#
+        .SYNOPSIS
+            Test-DbaSqlLoginWeakPassword finds any logins on SQL instance that are SQL Logins and have a password that is either null or same as the login
+
+        .DESCRIPTION
+            The purpose of this function is to find SQL Server logins that have no password or the same password as login. You can add your own password to check for or add them to a csv file.
+            By default it will test for empty password and the same password as username.
+
+        .PARAMETER SqlInstance
+            The SQL Server instance you're checking logins on. You must have sysadmin access and server version must be SQL Server version 2008 or higher.
+
+        .PARAMETER SqlCredential
+            Allows you to login to servers using SQL Logins instead of Windows Authentication (AKA Integrated or Trusted). To use:
+
+            $scred = Get-Credential, then pass $scred object to the -SqlCredential parameter.
+
+            Windows Authentication will be used if SqlCredential is not specified. SQL Server does not accept Windows credentials being passed as credentials.
+
+            To connect as a different Windows user, run PowerShell as that user.
+
+        .PARAMETER SqlPass
+            Specifies a list of passwords to include in the test for weak passwords.
+
+        .PARAMETER Path
+            Specifies a csv file path of passwords to include in the test for weak passwords.
```

Comment:
This should be removed and simply have the command handle multiple logins. The user would pass the data to the function via whatever method they use to get data out of the CSV file, or other sources.

If that makes sense?

Reply:
`DictionaryPath`?

Reply:
Oh, I think the path itself and functionality should be removed...not just renaming the parameter.

Reply:
So if I understand correct. The command should be able to handle that you do import-csv file.csv | test-dbasqlloginweakpassword ? 
this instead of having the function beeing able to read the csv file?

It will make the function more what it should do. I'm just not sure on how to make it accept through the pipeline when you have that option allreary on one parameter?